other/bootstrap_daemon/tox-bootstrapd.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Tox DHT Bootstrap Daemon
+After=network.target
+
+[Service]
+Type=forking
+PermissionsStartOnly=true
+ExecStartPre=-/bin/mkdir /var/run/tox-bootstrapd -p
+ExecStartPre=/bin/chown tox-bootstrapd:tox-bootstrapd -R /var/run/tox-bootstrapd
+WorkingDirectory=/var/lib/tox-bootstrapd
+ExecStart=/usr/bin/tox-bootstrapd /etc/tox-bootstrapd.conf
```

Comment:
There is a convention that everything built from source (i.e. ` make && make install`) ends up at `/usr/local/`, so I would replace this line with
```sh
ExecStart=/usr/local/bin/tox-bootstrapd /etc/tox-bootstrapd.conf
```
This is also what the .sh script is using.

Why are you using `/usr/`? Did the daemon get installed there when you built it from source?